action.yml

```diff
@@ -8,15 +8,17 @@ inputs:
   circleci-token:
     description: 'A personal API token to CircleCI'
     required: true
-  triggered_key:
-    description: 'A uuid specific for this job added by user'
+  kwargs:
+    description: 'json of extra arguments to pass to the workflow'
     required: false
+    default: '{}'
 runs:
   using: "composite"
   steps:
-    - run: ${{ github.action_path }}/trigger_circleci_pipeline.sh ${{ inputs.triggered-workflow }} ${{ inputs.triggered_key }}
+    - run: ${{ github.action_path }}/trigger_circleci_pipeline.sh ${{ inputs.triggered-workflow }}
```

Comment:
Do we want to pass the triggered workflow also as env var to avoid shell injection?